src/forms/InputTime.jsx

```diff
@@ -0,0 +1,120 @@
+// @flow
+import * as React from 'react';
+import cx from 'classnames';
+
+import withErrorList from '../utils/components/withErrorList';
+import FieldsetTime from './FieldsetTime';
+
+type Props = React.ElementConfig<HTMLInputElement> & {
```

Comment:
Could you add Docgen comments for these props? You can see an example in [this PR](https://github.com/meetup/meetup-web-components/pull/598/files#diff-8c4ea546d2c1ed928f5f523260d0001cR19)